ydb/core/tx/schemeshard/schemeshard__init_root.cpp

```diff
@@ -69,7 +69,7 @@ struct TSchemeShard::TTxInitRoot : public TSchemeShard::TRwTxBase {
             auto response = Self->LoginProvider.CreateGroup({
                 .Group = defaultGroup.GetName(),
                 .Options = {
-                    .CheckName = false
+                    .StrongCheckName = false
```

Comment:
You changed the semantics of this flag. As I understand it, you want to check names with different complexity. Then you should create new `option`